src/main/web/src/apps/ontology/app-components/method-details-view/methodDetails.html

```diff
@@ -49,7 +49,7 @@ <h2 class="om-panel-title underlined-heading">
 				<div ng-if="!isSystemMethod">
 					<div ng-hide="editing || confirmationNecessary">
 						<button class="btn btn-primary" ng-click="editMethod($event)">{{'button.edit' | translate}}</button>
-
+						<button ng-show="selectedVariable" class="btn btn-secondary" ng-click="showSelectedVariable($event)">{{'button.back' | translate}}</button>
```

Comment:
Improvement: When PSM panel is opened from the variable panel, close button should redirect back to the variable panel.